### DISCUSSION TITLE
Use FetchXMLBuilder as an Edge extension

### DISCUSSION THREAD

**Discussion by mlathouwers** [rappen/FetchXMLBuilder#764] [Ideas]
A consultants we service a lot of customers. We use seperate Edge profiles to have a good SSO experience. 
It would be great to be able to open one of the edge profiles and be able to use FetchXMLBuilder within that experience, without setting up the connection, as it is validated already.

**Comment by rappen** [comment]
I guess this idea would be a feature in XrmToolBox, not just in FetchXML Builder...
Any thoughts about this, mr @mscrmtools ?

**Reply by BetimBeja** [reply]
Maybe some input from @rajyraman and @melamriD365 who have already built chrome/edge extensions would be nice. I think getting a XrmToolBox2.0 as a browser extension would be really great... All tools would need to be rewritten in Javascript/Typescript though... and managing multiple-connection tools would be difficult. I want to be part of this if someone takes the responsibility to set it up 😄

**Reply by MscrmTools** [reply]
I will never start something like this. Too much work and I'm not a web developer.
If someone wants to start this kind of project, do it. I could eventually provide some of my tools with this new architecture but not more
